### DISCUSSION TITLE
Continuation of script problem.

### DISCUSSION THREAD

**Discussion by KoenBouten** [Set-OutlookSignatures/Set-OutlookSignatures#74] [General]
Hello Markus,

Here you have the text file with the output of the 3.6.0 script which you asked for last time.

But there was one thing I found in the meantime:

3.3.0 = line 748
3.6.0 = line 744
The script displays things in a different order/way at this part.
(domainD.local is a one-way-domain-trust & domainH.nl is a forrest-trust)

I hope this will be enough information.

[Script output V3.6.txt](https://github.com/GruberMarkus/Set-OutlookSignatures/files/11352226/Script.output.V3.6.txt)
[Script OutputV3.3.txt](https://github.com/GruberMarkus/Set-OutlookSignatures/files/11352255/Script.OutputV3.3.txt)

**Comment by GruberMarkus** [comment]
Closing, as this discussion has been converted to issue #75.
